ansible/tasks/setup-wal-g.yml

```diff
@@ -86,20 +21,45 @@
     mode: '0770'
   when: nixpkg_mode
 
-- name: Install wal-g from nix binary cache
+- name: Install wal-g 2 from nix binary cache
   become: yes
   shell: |
-    sudo -u wal-g bash -c ". /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh && nix profile install github:supabase/postgres/{{ git_commit_sha }}#wal-g"
+    sudo -u wal-g bash -c ". /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh && nix profile install github:supabase/postgres/{{ git_commit_sha }}#wal-g-2"
+  when: stage2_nix
+
+- name: Install wal-g 3 from nix binary cache
+  become: yes
+  shell: |
+    sudo -u wal-g bash -c ". /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh && nix profile install github:supabase/postgres/{{ git_commit_sha }}#wal-g-3"
   when: stage2_nix
 
 - name: Create symlink for wal-g from Nix profile to /usr/local/bin
   ansible.builtin.file:
-    src: /home/wal-g/.nix-profile/bin/wal-g
-    dest: /usr/local/bin/wal-g
+    src: /home/wal-g/.nix-profile/bin/wal-g-2
+    dest: /usr/local/bin/wal-g-v2
```

Comment:
We'd want to keep the old version backwards compatible, unfortunately

Reply:
@pcnc so there you mean old version needs to be `/usr/local/bin/wal-g`

Reply:
@pcnc maybe it is just that v2 needs to be installed at /usr/local/bin/wal-g but v3 installed at /usr/local/bin/wal-g-v3  but the default that can be run under command  wal-g in new ami is the v3 version is that the gist of what is needed

Reply:
I @pcnc I think I got this resolved in the way you needed